README.md

```diff
@@ -110,6 +110,14 @@ Tasks and features still discussed to be done:
   value;
 - support SHA512 if that would be ever needed.
 
+### Development
+
+Use `danger_no_encryption` Rust flag to disable data encryption for the debug purposes. E.g.:
```

Comment:
I'd call it `dangerous_disable_encryption` instead.

Reply:
This one sounds less clunky, thanks!